README.md

```diff
@@ -31,6 +31,10 @@ Modules in `.spec.js` and `.test.js` are added to `devDependencies`
 
 `--yarn`    Use [yarn](https://yarnpkg.com) instead of npm
 
+`--install-types`  Tries to install `@types` type definitions for modules that don't have them packaged
+
+`--include "<path>"`  Specify which glob pattern should be watched for changes. (**Double quotes are required to avoid expanding the path**)
```

Comment:
can we also skip this flag to assume .js, jsx, ts, tsx files are watched

Reply:
Sorry, I don't think I understand.
By default, ts and is files are watched, but, the user might want to watch only a single file or folder, for example.

Reply:
I see. I usually don't like adding feature flags unless they have a very obvious and strong case for them. In this case, I don't see it because you'd need the tool to run in the background silently and install dependencies at a project level.

Reply:
Hm... I understand.
So should we not allow a custom watch glob pattern?
Maybe there's some "examples" folder the user does not want to install dependencies from?
I'm OK with removing it, since we're already watching TS files by default now, but that is just my use case

Reply:
Yep, I'd like to skip it

Reply:
Ok, I'll remove it then

Reply:
Done. 
I've kept the whole `includePath` variable structure because it makes it easier to change it in the future (and I have the feeling that will be necessary).

The flag itself is gone, tho. The watch glob can only be changed by changing source now.